webhook/mutation/pod_mutator_test.go

```diff
@@ -314,7 +341,16 @@ func TestUseImmutableImage(t *testing.T) {
 				&corev1.Namespace{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:   "test-namespace",
-						Labels: map[string]string{mapper.InstanceLabel: "dynakube"},
+						Labels: map[string]string{mapper.InstanceLabel: dynakubeName},
+					},
+				},
+				&corev1.Secret{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      dynakubeName,
+						Namespace: "dynatrace",
+					},
+					Data: map[string][]byte{
+						dtclient.DynatraceDataIngestToken: []byte(dataIngestToken),
 					},
```

Comment:
Could add function like `prepareDynakubeSecret()` since this is used a lot of times.

Reply:
function added